acceptance/setup/pre_suite/40_install_deps.rb

```diff
@@ -47,7 +47,11 @@
       if test_config[:install_type] == :git then
         case os
         when :debian
-          on database, "apt-get install -y --force-yes openjdk-7-jre-headless rake unzip"
+          on database, "apt-get install -y rake unzip"
```

Comment:
I wanted to leave off the force-yes because I don't know why it's there, and I suspect it shouldn't be (I'd planned to add it back if the install doesn't work without it.)